instantsearch.js/facet-dropdown/src/Dropdown.js

```diff
@@ -90,7 +90,7 @@ export function createDropdown(
       setTimeout(() => {
         state.windowClickListener = event => {
           // Close if the outside is clicked
-          if (!rootElem.contains(event.target)) {
+          if (!event.composedPath().includes(rootElem)) {
```

Comment:
[composedPath](https://caniuse.com/?search=composedPath) does not work on IE but as it's userland code I suppose it's okay